src/azure-cli/HISTORY.rst

```diff
@@ -43,6 +43,11 @@ Release History
 * az deployment {group/mg/sub/tenant} list: Support provisioningState filtering
 * az deployment: Fix the parse bug for comment under the last argument
 
+**ARO**
+
+* Added new command 'aro' for Azure RedHat OpenShift v4
+* Added support create, delete, list, show, update, list-credentials commands
+
```

Comment:
Don't have to list history in this file, besides this file is not mean to modify by developer itself.

The content of this file comes from every PR title and PR body since last release. 

Please take a look this [doc](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests)